Thumbprint/Targets/Thumbprint/Components/Pill.swift

```diff
@@ -15,27 +15,23 @@ open class Pill: UIView, UIContentSizeCategoryAdjusting {
         public static var defaultHeight: CGFloat = 24.0
         public static var defaultPadding: CGFloat = 12.0
         public static var defaultTextStyle: Font.TextStyle = .title7
-        public static var defaultIconSize = Icon.Size.tiny
 
         var height: CGFloat
         var padding: CGFloat
         var textStyle: Font.TextStyle
-        var iconSize: Icon.Size
 
         public init(height: CGFloat = defaultHeight,
                     padding: CGFloat = defaultPadding,
-                    textStyle: Font.TextStyle = defaultTextStyle,
-                    iconSize: Icon.Size = defaultIconSize) {
+                    textStyle: Font.TextStyle = defaultTextStyle) {
             self.height = height
             self.padding = padding
             self.textStyle = textStyle
-            self.iconSize = iconSize
         }
     }
 
     // MARK: - Implementation Views
 
-    private let iconImageView: UIImageView = {
+    public let iconImageView: UIImageView = {
```

Comment:
QQ - are we exposing the UIImageView (as opposed to the UIImage) to allow the caller to change the positioning / constraints, etc?

Reply:
Yes